app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
-    <string name="pref_zoom_max">Maximum Zoom</string>
-    <string name="pref_clipboard_sync">Clipboard Sync</string>
+    <string name="pref_zoom_min">Min</string>
+    <string name="pref_zoom_max">Max</string>
+    <string name="pref_clipboard_sync">Clipboard sync</string>
     <string name="pref_credential">Credentials</string>
-    <string name="pref_cred_autocomplete">Enable Autocomplete</string>
-    <string name="pref_cred_autocomplete_summary">Suggests known credentials when connecting to a new Server</string>
+    <string name="pref_cred_autocomplete">Auto-completion</string>
+    <string name="pref_cred_autocomplete_summary">Suggest saved logins when connecting to new servers</string>
     <string name="pref_discovery">Discovery</string>
-    <string name="pref_discovery_timeout">Timeout (Seconds)</string>
-    <string name="pref_discovery_auto_start">Auto Start</string>
-    <string name="pref_discovery_auto_start_summary">Start Discovery when App starts</string>
-    <string name="pref_discovery_restart">Restart</string>
-    <string name="pref_discovery_restart_summary">Restart Discovery after it stops</string>
-    <string name="pref_discovery_restart_delay">Restart Delay (Minutes)</string>
-    <string name="pref_import_export">Import/Export Servers</string>
+    <string name="pref_discovery_timeout">Timeout (sec)</string>
+    <string name="pref_discovery_auto_start">Auto-start</string>
+    <string name="pref_discovery_auto_start_summary">Discover servers when opening the app</string>
+    <string name="pref_discovery_restart">Always look for new servers</string>
+    <string name="pref_discovery_restart_summary">Keep running server discovery</string>
+    <string name="pref_discovery_restart_delay">Restart delay (min)</string>
+    <string name="pref_import_export">Import/export servers</string>
     <string name="pref_debugging">Debugging</string>
     <string name="pref_logs">Logs</string>
-    <string name="pref_keycode_map">Keycode Map</string>
-    <string name="pref_experimental_toolbar_swipe_close">Swipe to close Toolbar Drawer</string>
-    <string name="pref_experimental_toolbar_swipe_close_summary">During a remote session, you can swipe to close currently open Toolbar</string>
-    <string name="pref_experimental_indicator">Discovery indicator for saved Servers</string>
-    <string name="pref_experimental_indicator_summary">Highlights already saved servers which are also found by Discovery.\nRequires App restart.</string>
-    <string name="pref_experimental_immersive_mode">Full immersive mode</string>
-    <string name="pref_experimental_immersive_mode_summary">Hides all system bars in fullscreen mode</string>
+    <string name="pref_keycode_map">Keycode map</string>
+    <string name="pref_experimental_toolbar_swipe_close">Swipe to close toolbar drawer</string>
+    <string name="pref_experimental_toolbar_swipe_close_summary">Swipe during a remote sessions to exit the toolbar</string>
```

Comment:
You should leave these two as it is. They are going to be removed in next version.